www/content/attributes/hx-trigger.md

```diff
@@ -56,7 +56,7 @@ is seen again it will reset the delay.
 is seen again before the delay completes, it is ignored, the element will trigger at the end of the delay.
 * `from:<Extended CSS selector>` - allows the event that triggers a request to come from another element in the document (e.g. listening to a key event on the body, to support hot keys)
   * A standard CSS selector resolves to all elements matching that selector. Thus, `from:input` would listen on every input on the page.
-  * The CSS selector is only evaluated once and is not re-evaluated when the page changes. If you need to detect dynamically added elements use an event filter, for example `click[event.target.matches('input')]`
+  * The CSS selector is only evaluated once and is not re-evaluated when the page changes. If you need to detect dynamically added elements use a [standard event filter](#standard-event-filters), for example `hx-trigger="click[event.target.matches('button')] from:body"` which would listen on every button on the page.
```

Comment:
If you allow me a last nitpicking, technically this listens on the body and not the buttons _(the `addEventListener` is on the body)_, but it will... catch? intercept? (those are the 2 words that come to mind but there may be a better one to describe it) the button `click` events.

I don't think we need to explain what event bubbling here is (which could make the sentence too verbose), but maybe we can just insert a link on the verb you will choose, that points to the [MDN event bubbling doc](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Event_bubbling) ?